templates/treat_ad_long.yaml

```diff
@@ -0,0 +1,908 @@
+- name: TREAT-AD Internal Sharing Site
+  type: Project
+  acl:
+  - principal_id: 3425650
```

Comment:
nit (minor): Did you want to leave comments for who these principals are to improve readability for yourself?  
```
  - principal_id: 3425650 # user 1
```

Reply:
I love that idea.  I have no idea who those are by looking at the number.  Thanks for the feedback, Tom!